File: Android/CarBuds/app/src/main/java/com/ali/cs491/carbuds/MainActivity.java
package com.ali.cs491.carbuds;

import android.content.Context;
import android.content.Intent;
import android.content.SharedPreferences;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    public static String MYPREFERENCES = "myprefs";
    // Used to load the 'native-lib' library on application startup.
    static {
        System.loadLibrary("native-lib");
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
      //  SharedPreferences sharedpreferences = getSharedPreferences(MYPREFERENCES, Context.MODE_PRIVATE);
      //  int isLogged = sharedpreferences.getInt("login", 0);

        // Example of a call to a native method
        Button loginButton = findViewById(R.id.loginButton);
        Button signUpButton = findViewById(R.id.signUpButton);
        //  to do add color for buttons and background

        loginButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, LoginActivity.class);
                startActivity(intent);
            }
        });
        signUpButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
             //   Intent intent = new Intent(MainActivity.this, SignUpActivity.class);
           //     startActivity(intent);
            }
        });

    }


}

File: Android/CarBuds/CarBuds/app/src/main/java/com/ali/cs491/carbuds/MainActivity.kt
package com.ali.cs491.carbuds

import android.content.Intent
import android.graphics.Color
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        loginButton.text = "Login"

        signupButton.text = "SignUp"

        loginButton.setOnClickListener {
            val intent = Intent(this, LoginActivity :: class.java )
            startActivity(intent)
        }
        signupButton.setOnClickListener {
            val intent = Intent(this, SignupActivity :: class.java )
            startActivity(intent)
        }
        loginButton.setBackgroundColor(Color.GREEN)
        signupButton.setBackgroundColor(Color.RED)
    }
}
